mappings/net/minecraft/block/BambooBlock.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_romdhvev net/minecraft/block/BambooBlock
```

Comment:
Did we lose the bamboo block mapping?!?!?

Reply:
yes I was confused about that
most of it directly copied

Reply:
sounds good!